src/main/xar-resources/data/configuration.xml

```diff
@@ -423,19 +415,54 @@
                         <para>Controls the indexing process.</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term>&lt;scheduler&gt;</term>
+                    <listitem>
+                        <para>Job scheduler for system or user jobs. </para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>&lt;parser&gt;</term>
+                    <listitem>
+                        <para>Default settings for parsing structured documents.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>&lt;serializer&gt;</term>
+                    <listitem>
+                        <para>Default settings for the serializer (external data representation).</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>&lt;transformer&gt;</term>
+                    <listitem>
+                        <para>Default settings for the XSLT Transformer.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>&lt;validation&gt;</term>
+                    <listitem>
+                        <para>Settings for XML validation.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>&lt;xquery&gt;</term>
+                    <listitem>
+                        <para>Define (built-in) modules that contain xQuery functions.</para>
+                    </listitem>
+                </varlistentry>
                 <varlistentry>
                     <term>&lt;xupdate&gt;</term>
                     <listitem>
                         <para>Configuration options related to XUpdate processing.</para>
                     </listitem>
                 </varlistentry>
             </variablelist>
-            <para>The following sections describe the attributes and child elements of the above
-                elements.</para>
+            <para>The following sections describe the most commonly modified attributes and child elements of the above elements, including how to change the default behavior of exist-db's handling of whitespace characters.</para>
             <section>
                 <title>&lt;db-connection&gt;</title>
                 <para>This element contains basic default storage settings for eXist-db, including
-                    memory and system limits. Only one <sgmltag>db-connection</sgmltag> should be
+                    memory and system limits. Only one &lt;db-connection&gt; should be
```

Comment:
Was there a problem with [`<sgmltag>`](http://tdg.docbook.org/tdg/4.5/sgmltag.html)? Seems like the semantically appropriate approach for inline tags in DocBook 4.5; DocBook 5 moved this to [`<tag>`](http://tdg.docbook.org/tdg/5.0/tag.html).

Reply:
well there is display problem with xml elements in section titles, which is not related to the docbook markup. My reason for switching was to increase consistency. Once we update the docbook markup for all contents, we might very well land on `<tag>` or `<sgmltag>`, but until then we should at least stick to one or the other on a per document basis

Reply:
If we were consistent in other documents, I'd suggest we limit this change to titles—limiting our need for parsing string contents in a future pass. But if we use escaped angle brackets elsewhere for inline, then I guess we're inconsistent already and will need to do a pass on this anyway...